src/main/java/com/codessquad/qna/controller/PageController.java

```diff
@@ -0,0 +1,43 @@
+package com.codessquad.qna.controller;
+
+import com.codessquad.qna.domain.Question;
+import com.codessquad.qna.repository.QnaRepository;
+import com.codessquad.qna.web.PageUtil;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class PageController {
+
+  private static Logger log = LoggerFactory.getLogger(PageController.class);
+  private static final int PAGE_SIZE = 4;
+
+  @Autowired
+  private QnaRepository qnaRepository;
+
+  @GetMapping(value = "/")
+  public String index(Model model, Optional<Integer> page) {
```

Comment:
Optional<Integer> page로 받으신 이유가 있을까요?
저도 페이징을 자주 해보진 않았지만, 컨트롤러의 파라미터로 Optional을 사용하는 것도 일반적이지 않은 패턴으로 보입니다.
`@PageableDefault`, `Pageable` 관련해서 찾아보세요.

```suggestion
  public String index(Model model, Pageable pageable) {
```

그리고 Optional 관련해서 문서를 첨부했는데, 6번의 내용을 한번 참고해보세요.

http://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/

Reply:
@tramyu 새로운것을 알았네요! 감사합니다 !!